releasenotes/notes/cmd2-0.9.0-35b6c209cbea798f.yaml

```diff
@@ -0,0 +1,7 @@
+---
+issues:
+  - |
+    All OSA releases earlier than 17.0.5, 16.0.4, and 15.1.22 will fail to build
```

Comment:
In the RC branch patch, we should change this wording to 'All RPC-O releases earlier than ...'